config/samples/resources/dnsrecordset/dns-routing-policy-wrr-health-checks/resourcemanager_v1beta1_project.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Project
+metadata:
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: abandon
```

Comment:
Will defer all fixes for samples in a followup PR.

Reply:
What's the timeline of the followup PR? I think we want #1399 and the follow-up PR to be covered by the same release so that we won't have potentially failing/flaky samples test to reduce our confidence in release.